sirepo/job_supervisor.py

```diff
@@ -275,7 +275,7 @@ def _get_uids_and_files():
             for f in pkio.sorted_glob(_DB_DIR.join('*{}'.format(
                     sirepo.simulation_db.JSON_SUFFIX,
             ))):
-                n = sirepo.sim_data.uid_from_jid(f.purebasename)
+                n = sirepo.sim_data.split_jid(f.purebasename).uid
```

Comment:
This is an implicit coupling used for optimization. I think it might be better to document: jid=f.purebasename.

Reply:
https://github.com/radiasoft/sirepo/pull/2534/commits/9ce89be2fa5fb91607e6a20d55f8bf0cb9a69f45